### PR TITLE
LPS-199014 Show more info on error when copying to

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
When more than 10 items failed to copy: 

Screenshot 2023-10-18 at 12 27 47

When less than 10 items fail: 

Screenshot 2023-10-18 at 11 35 33

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
Right now it show this message when errors are more than 10

image


🔴 Need buildLang

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Review by marcogalluzzi — APPROVED** [review]

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/DLFolderSelector.js [hunk 1]
```diff
@@ -75,11 +75,35 @@ const DLFolderSelector = ({
...
+			Liferay.Language.get('x-items-could-not-be-copied'),
+			errorSize
+		)}
+			<ul class="mb-0 mt-2 pl-3">`;
+
+		errorMessages.forEach((message) => {
+			formattedMessage = `${formattedMessage}<li>${message}</li>`;
+		});
+
+		formattedMessage = `${formattedMessage}</ul>`;
```
[the remark this answers is not in the document]
Comment (ambrinchaudhary) [reply modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/DLFolderSelector.js]: Thanks @boton, you are right. I changed it, please review.

**Comment by ambrinchaudhary** [comment]
> In general LTGM but I found the template part a bit tricky I left an inline comment about it

Thanks @boton, you are right. I changed it, please review. 
Sorry @marcogalluzzi for requesting your review again, it is not needed

**Review by marcogalluzzi — APPROVED** [review]

**Review by boton — APPROVED** [review]
LGTM

**Comment by boton** [comment]
ci:forward

**Comment by ambrinchaudhary** [comment]
Hi @Jeremy-Ch This test [LocalFile.DocumentsandmediaUpgrade#FileUploadLimitIsPreservedAfterUpgradeFrom70105](https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/292763//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_DocumentsandmediaUpgrade_FileUploadLimitIsPreservedAfterUpgradeFrom70105_) failed twice but looking at PR changes this shouldn't be affected. Could you please confirm if it is related? Otherwise we can force forward. Thanks!

**Comment by Jeremy-Ch** [comment]
The failures are unrelated.

**Comment by Jeremy-Ch** [comment]
ci:forward:force

**Pull request closed without merging**